test/image-prepare

```diff
@@ -213,6 +213,7 @@ def main():
     parser = argparse.ArgumentParser(
         description='Prepare testing environment, download images and build and install cockpit',
         formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('-n', '--no-c', action='store_true', help='Skip installing the C code')
```

Comment:
Hmm, couldn't we just make `--python` do that? That will remove the /usr/bin/cockpit-bridge binary anyway, so the rpm won't get us anything.

Reply:
By default we install the C bridge and webpack parts anyway.  That was done so that if we make some changes to `cockpit-ws` or (more likely) `pkg/` we get to test those changes against the Python bridge, instead of using the old distro package.  That seems like a reasonable default and I'd like to keep that, even with `--python`.

Reply:
Hmm, I don't quite understand this TBH. We always want to install cockpit-ws and cockpit-session and such, I understood this option as "no C bridge", not "no C parts at all". Note that image-prepare isn't a tool for iterating (it's waay too slow for that)

Reply:
Anyway, if that were a permanent option I'd object harder, but as hopefully both --python and --no-c will go away in a few months, I can stomach this hack.

Reply:
This comment got dropped.